posthog-react-native/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Next
 
+# 3.4.1 - 2024-11-27
```

Comment:
Should this be a minor since it adds a new feature?

Reply:
@marandaneto  It's technically just for debugging ability, no plans to use in product right now. You know better here, so let me know what you think!

Reply:
eh I think @marandaneto has a good point since it's kinda going to be used in our product (once those events get propagated to the payload, all users will have access to it).  Useful for debugging on our end, sure, but could also be handy for customers I imagine.

Reply:
your call here, just food for though, the patch is just for bug fixes usually (semver).

Reply:
Makes sense for traditional semver, updated to minor bump!